client/my-sites/plugins/plugin-upload/index.jsx

```diff
@@ -203,9 +203,8 @@ const mapStateToProps = ( state ) => {
 	// before data has loaded.
 	const isEligible = isEligibleForAutomatedTransfer( state, siteId );
 	const isEligibleForHostingTrial =
-		isUserEligibleForFreeHostingTrial( state ) &&
-		site.plan.is_free &&
-		config.isEnabled( 'hosting-trial' );
+		isUserEligibleForFreeHostingTrial( state ) && site && site.plan?.is_free;
+	config.isEnabled( 'hosting-trial' );
```

Comment:
What's happening here? The trial check went over to a new line :) Can we also replace the site check with `site?.plan?.is_free`?